runhouse/resources/hardware/cluster.py

```diff
@@ -390,9 +391,21 @@ def from_name(
             _resolve_children=_resolve_children,
         )
         if cluster and cluster._creds and not dryrun:
-            from runhouse.resources.secrets.utils import _write_creds_to_local
+            from runhouse.resources.secrets import Secret
+            from runhouse.resources.secrets.provider_secrets.ssh_secret import SSHSecret
+
+            if isinstance(cluster._creds, SSHSecret):
+                cluster._creds.write(write_config=False)
```

Comment:
not sure if this is the right place to do it, but shouldn't we also save this info down to the local rh config?

Reply:
I guess just refactoring based on current behavior, when we write down cluster creds (e.g. sharing to another device) we don't save it to rh config. I think writing it down to the config as well would be fine, but this was mostly implemented to keep track of secrets to write/remove during login/logout, not sure if it makes a difference anywhere else.